doc/devel/contributing.rst

```diff
@@ -33,19 +64,20 @@ Issues for new contributors
 
 While any contributions are welcome, we have marked some issues as
 particularly suited for new contributors by the label
-`good first issue <https://github.com/matplotlib/matplotlib/labels/good%20first%20issue>`_
-These are well documented issues, that do not require a deep understanding of
-the internals of Matplotlib. The issues may additionally be tagged with a
-difficulty. ``Difficulty: Easy`` is suited for people with little Python experience.
-``Difficulty: Medium`` and ``Difficulty: Hard`` require more programming experience.
-This could be for a variety of reasons, among them, though not necessarily all at
-the same time:
+`good first issue <https://github
+.com/matplotlib/matplotlib/labels/good%20first%20issue>`_ These are well
+documented issues, that do not require a deep understanding of the internals
+of Matplotlib. The issues may additionally be tagged with a difficulty.
+``Difficulty: Easy`` is suited for people with little Python experience.
+``Difficulty: Medium`` and ``Difficulty: Hard`` require more programming
+experience. This could be for a variety of reasons, among them, though not
+necessarily all at the same time:
```

Comment:
```suggestion
particularly suited for new contributors by the label `good first issue
<https://github.com/matplotlib/matplotlib/labels/good%20first%20issue>`_. These
are well documented issues, that do not require a deep understanding of the
internals of Matplotlib. The issues may additionally be tagged with a
difficulty. ``Difficulty: Easy`` is suited for people with little Python
experience. ``Difficulty: Medium`` and ``Difficulty: Hard`` require more
programming experience. This could be for a variety of reasons, among them,
though not necessarily all at the same time:
```